cli/deduplifhirLib/normalization.py

```diff
@@ -247,6 +248,19 @@ def british_to_american(input_text):
     return input_text
 
 
+def normalize_date_text(input_text):
+    """
+    Normalizes the given date string
+
+    Arguments:
+        input_text: the input date text to normalize
+    
+    Returns:
+        The normalized date string
+    """
+    d = date_parser.parse(input_text)
+    return d.strftime("%Y-%m-%d")
```

Comment:
Should we handle exceptions here if it fails to parse? Not sure we'd run into it later, but what currently happens if someone has "TEST" in a birth date column?

Reply:
I added an exception handle there to handle invalid date formats.